test/integration/targets/vmware_guest_register_operation/tasks/main.yml

```diff
@@ -0,0 +1,146 @@
+# Test code for the vmware_guest_register_operation module
+# Copyright: (c) 2019, sky-joker <sky.jokerxx@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- import_role:
+    name: prepare_vmware_tests
+  vars:
+    setup_attach_host: true
+    setup_datastore: true
+    setup_cluster: true
+    setup_virtualmachines: true
```

Comment:
add `setup_resource_pool: true`, this will create a resource pool called `DC0_C0_RP1`.